src/XRP/issued-currency-client.ts

```diff
@@ -531,6 +531,32 @@ export default class IssuedCurrencyClient {
     )
   }
 
+  /**
+   * Re-enables rippling on the trust line between this account (issuing account) and another account.
+   * Note that the trust line's limit is set to 0.
+   *
+   * @see https://xrpl.org/freezes.html#enabling-or-disabling-individual-freeze
+   *
+   * @param trustLinePeerAccount The X-Address of the account involved in the trust line to re-enable rippling.
+   * @param currencyName The currency of the trust line to re-enable rippling.
+   * @amount The maximum amount of debt to allow on this trust line.
+   * @param wallet The wallet re-enabling rippling on the trust line.
+   */
+  public async clearNoRipple(
```

Comment:
I think this name is a bit confusing - I would suggest something like `enableRipple`, but that's the DefaultRipple flag (maybe that should be changed to `enableDefaultRippling`?)

Reply:
i tried `enableTrustLineRippling` - let me know what you think. i'll go back and change `setNoRipple` to `disableTrustLineRippling` if we like this naming